src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/SuperCloneCheck.java

```diff
@@ -37,7 +37,32 @@
  * <pre>
  * &lt;module name=&quot;SuperClone&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class A {
+ *
+ *  public Object clone() {
+ *   return super.clone(); // OK
+ *  }
+ * }
+ *
+ * class B {
+ * private int b;
  *
+ *  public B clone() {
+ *   B other = new B(); // violation, does not call super.clone()
```

Comment:
Violation comment is misplaced, move it to previy line. Same for other comments.